monkey/monkey_island/cc/models/monkey.py

```diff
@@ -68,6 +81,10 @@ def get_os(self):
             os = "windows"
         return os
 
+    def renew_ttl(self, duration=DEFAULT_MONKEY_TTL_EXPIRY_DURATION_IN_SECONDS):
+        logger.debug("Renewing TTL for monkey {0}".format(self.guid))
```

Comment:
I don't think this is relevant info for us.

Reply:
Deleted.